src/client/containers/layout.js

```diff
@@ -120,6 +121,10 @@ class Layout extends React.Component {
 						<FontAwesomeIcon icon={faLink}/>
 						{' Relationship Types '}
 					</NavDropdown.Item>
+					<NavDropdown.Item href="/identifier-types">
+						<FontAwesomeIcon icon={faBarcode}/>
```

Comment:
Seems a bit out of scope for this PR, but could you please add the `fixedWidth` attribute for all the icons in this menu?
https://origin.fontawesome.com/v5/docs/web/use-with/react#fixed-width-icons

Before:
<img width="222" alt="image" src="https://github.com/metabrainz/bookbrainz-site/assets/6179856/87bb7c45-fc30-4bfa-9fef-c2301ae77237">


After: 
<img width="226" alt="image" src="https://github.com/metabrainz/bookbrainz-site/assets/6179856/70b7d8f0-4c7d-44d0-9817-ccc22879c81b">